scalableminds/webknossos
# Remove team assignments from task

### Mailable description of changes:
 - Tasks no longer have the field 'team'. 
 - The linkage between Task and Team is implemented through Project. (The Task knows the Project and the Projects knows the Team)
 - This leads to the following changes:
   The creation on Tasks does not require the name of the Team anymore.
   An example for Bulk-Creation now looks like this:
   `ROI2017_wkw, 5afeb79073000071006904a5, sampleExp, 1, 512,512,512, 0, 0, 0, 1, 500, 500, 500, 100, 100, 100, sampleProject`
    instead of 
`ROI2017_wkw, 5afeb79073000071006904a5, sampleExp, 1, 512,512,512, 0, 0, 0, 1, Connectomics department, 500, 500, 500, 100, 100, 100, sampleProject
`

### Steps to test:
test if the basic functions for 'task' are still working
- create a new task
- press the 'Get a new Task'-Button

test the project_list_view:
- go to 'Administration' -> 'Projects' and verify that the names of the projects are shown

### Issues:
- fixes #1279 

------
